src/chapters/quickconnect.xml

```diff
@@ -0,0 +1,90 @@
+        <para>The quickconnect extension is packaged as a <filename>.tar.gz</filename> file containing
+            only the extension itself, <filename>guacamole-auth-quickconnect-0.9.14.jar</filename>, which must
+            ultimately be placed in <filename>GUACAMOLE_HOME/extensions</filename>.</para>
+    </section>
+    <section xml:id="installing-quickconnect">
+        <title>Installing the quickconnect extension</title>
+        <para>Guacamole extensions are self-contained <filename>.jar</filename> files which are
+            located within the <filename>GUACAMOLE_HOME/extensions</filename> directory.
+                <emphasis>If you are unsure where <varname>GUACAMOLE_HOME</varname> is located on
+                your system, please consult <xref linkend="configuring-guacamole"/> before
+                proceeding.</emphasis></para>
+        <para>To install the extension, you must:</para>
+        <procedure>
+            <step>
+                <para>Create the <filename>GUACAMOLE_HOME/extensions</filename> directory, if it
+                    does not already exist.</para>
+            </step>
+            <step>
+                <para>Place the <filename>guacamole-auth-quickconnect-0.9.14.jar</filename> file in
+                    the <filename>GUACAMOLE_HOME/extensions</filename> directory.</para>
+            </step>
+        </procedure>
+        <section xml:id="guac-quickconnect-config">
+            <title>Configuring Guacamole for the quickconnect extension</title>
+            <indexterm>
+                <primary>configuring quickconnect</primary>
+            </indexterm>
+            <para>This module has no configuration options.</para>
+        </section>
+        <section xml:id="completing-quickconnect-install">
+            <title>Completing the installation</title>
+            <para>Guacamole will only load newly-installed extensions during startup, so your
+                servlet container will need to be restarted before the quickconnect extension
+                can be used. <emphasis>Doing this will disconnect all active users, so be sure
+                    that it is safe to do so prior to attempting installation.</emphasis>
+                When ready, restart your servlet container and give the extension a try.</para>
+        </section>
+    </section>
+    <section xml:id="using-quickconnect">
+        <title>Using the quickconnect extension</title>
+        <para>The quickconnect extension provides a field on the home page that allows you to enter
+             a Uniform Resource Identifier (URI) to create a connection.  A URI is in the form:</para>
+            <informalexample><replaceable>protocol</replaceable>://<replaceable>username</replaceable>
+            :<replaceable>password</replaceable>@<replaceable>host</replaceable>:
+            <replaceable>port</replaceable>/?<replaceable>options</replaceable></informalexample>
```

Comment:
The documentation for the supported protocols refers to these as "parameters", not "options". I suggest:

* Consistently referencing the connection parameters as parameters
* Providing a concrete example or set of examples which demonstrates the use of these parameters
* Explaining, at least briefly, how multiple parameters are represented in the URI (`name1=value1&name2=value2&...`), as that may not be clear.

Reply:
Okay, I changed all of the appropriate references to parameters to use parameter instead of option, so that should be good.  Also provided three examples of URIs, including a RDP one with several parameters specified in the URI.